src/esp/gfx/Renderer.cpp

```diff
@@ -44,15 +45,14 @@ struct Renderer::Impl {
     }
   }
 
-  void draw(sensor::VisualSensor& visualSensor,
+  void draw(esp::sensor::CameraSensor& cameraSensor,
```

Comment:
What is the reason to change the VisualSensor to a CameraSensor?

Reply:
Thanks for the feedback, I was definitely not aware of all the uses of virtual functions in cpp, and have learned a lot! This is fixed now.